### ISSUE TITLE
AppCrash with MediaPlayer

### ISSUE THREAD

**Issue by rido-min** [microsoft/Windows-appsample-Xaml-Hosting#3]
When I lunch the app and select "MediaElement" it crashes with:

```
Fault bucket , type 0
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: XamlHostingSample.exe
P2: 0.0.0.0
P3: 5bbe9b42
P4: combase.dll
P5: 10.0.18254.1000
P6: aa72a7b2
P7: 80070057
P8: 0000000000091c32
P9: 
P10: 
```

```
Faulting application name: XamlHostingSample.exe, version: 0.0.0.0, time stamp: 0x5bbe9b42
Faulting module name: Windows.UI.Xaml.dll, version: 10.0.18254.1000, time stamp: 0xb74abca6
Exception code: 0xc000027b
Fault offset: 0x0000000000733852
Faulting process id: 0x37ac
Faulting application start time: 0x01d460fb90e47af4
Faulting application path: C:\code\Windows-appsample-Xaml-Hosting\x64\Debug\XamlHostingSample.exe
Faulting module path: C:\Windows\System32\Windows.UI.Xaml.dll
Report Id: 595f5bc8-a3a2-4684-bc54-92be6d699c77
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by bmitchell287** [comment]
@rido-min -Hi Ricardo, looking into this. For verification you're still using:
-OS version -17673
-VS 15.9 Preview 3
-Latest Windows Insider SDK

And to confirm,  you don't have the ability to select a podcast from the combo box and play a clip? It crashes when you click on the Media Player module?

**Comment by rido-min** [comment]
not exactly:
- OS Version: 10.0.18256.1000
- VS 15.9 Preview 4 (2828.3051)
- SDK 17763

Correct, I can't even click on the podcast cobo box because the app crashes before.

```
Unhandled exception at 0x00007FFCF5ED3BD2 (Windows.UI.Xaml.dll) in XamlHostingSample.exe: 0xC000027B: An application-internal exception has occurred (parameters: 0x0000028262D082B0, 0x0000000000000002). occurred

KernelBase.dll!00007ffd0491e33f()
combase.dll!00007ffd064b3cf0()
Windows.UI.Xaml.dll!00007ffcf5ed3bd2()
Windows.UI.Xaml.dll!00007ffcf5ed3dc5()
Windows.UI.Xaml.dll!00007ffcf5beb213()
Windows.UI.Xaml.dll!00007ffcf58a9f93()
Windows.UI.Xaml.dll!00007ffcf58a9754()
Windows.UI.Xaml.dll!00007ffcf59e156b()
Windows.UI.Xaml.dll!00007ffcf59e146c()
CoreMessaging.dll!00007ffd020772bc()
CoreMessaging.dll!00007ffd0209881d()
CoreMessaging.dll!00007ffd0207be30()
CoreMessaging.dll!00007ffd0207af50()
CoreMessaging.dll!00007ffd02079e55()
CoreMessaging.dll!00007ffd02079c8c()
user32.dll!00007ffd06f9ca66()
user32.dll!00007ffd06f9c582()
XamlHostingSample.exe!WinMain(HINSTANCE__ * hInstance, HINSTANCE__ * hPrevInstance, char * lpCmdLine, int nCmdShow) Line 113
	at c:\code\windows-appsample-xaml-hosting\xamlhostingsample\main.cpp(113)
[External Code]

```

**Comment by bmitchell287** [comment]
@rido-min Hi, we'll keep looking into this. We're closing the bug because we can't reproduce in our environment using OS/SDK versions 17763, or the current released version of VS 15.8.8. Again we'll keep looking into this, thanks for submitting.

**Comment by takazm** [comment]
This issue still persist and i don't think it has anything to do with the version of VS as i am getting that in release mode after deploying to a non-development machine.
